vm/chef/cookbooks/piwik/recipes/default.rb

```diff
@@ -39,6 +39,28 @@
   command 'tar xzf matomo.tar.gz -C /var/www/html --strip-components 1'
 end
 
+# Preparing Piwik
+bash 'configure_piwik_for_first_use' do
+  user 'root'
+  code <<-EOH
+    rm -Rf /var/www/html/plugins/Morpheus/icons/submodules
+    chown -R www-data:www-data /var/www/html
```

Comment:
use environment attribute for not repeating www-data. 

https://github.com/GoogleCloudPlatform/click-to-deploy/blob/0da175fb576302ff6d90cdf7fe74ddbb38c0b1ed/vm/chef/cookbooks/redmine/recipes/default.rb#L41